compiler/noirc_frontend/src/hir/comptime/interpreter/builtin.rs

```diff
@@ -200,26 +245,25 @@ fn as_slice(
 
 fn slice_push_back(
     interner: &NodeInterner,
-    mut arguments: Vec<(Value, Location)>,
+    arguments: Vec<(Value, Location)>,
     location: Location,
 ) -> IResult<Value> {
-    check_argument_count(2, &arguments, location)?;
+    let ((slice, _), (element, _)) = check_two_arguments(arguments, location)?;
 
-    let (element, _) = arguments.pop().unwrap();
-    let (mut values, typ) = get_slice(interner, arguments.pop().unwrap().0, location)?;
+    let (mut values, typ) = get_slice(interner, slice, location)?;
```

Comment:
Not very important, but I wonder if `location` here should be that of the `slice` argument (the `_` after `slice` in line 251)